### ISSUE TITLE
Add details on how to estimate related problems

### ISSUE THREAD

**Issue by bencrts** [malb/lattice-estimator#54]
Based on some recent issues/questions I've seen, perhaps we could consider adding some documentation on how to estimate related problems. In particular I am thinking about MLWE/LWR problems, etc. This is kind-of implicit in `schemes.py` when we add parameters for the schemes based on these problems, but it could be nice to expand a little bit.

@malb if you think it's worth doing I can take care of creating something.

**Comment by malb** [comment]
Yeah, I think that's a good idea, but I'd keep it at the documentation level
